### PR TITLE
[Launcher]: Refine Models

### PR DESCRIPTION
**Pull request by yqwang-ms**

### CODE REVIEW

File: frameworklauncher/src/test/resources/TestExpects/LauncherConfiguration.yml [hunk 1]
```diff
@@ -2,30 +2,17 @@
 amAllowNoneGpuJobOnGpuNode: false
 amAttemptFailuresValidityIntervalSec: 0
 amAttemptMaxCount: 0
+amCandidateNodesFactor: 0
```
Comment (qinchen123) [inline frameworklauncher/src/test/resources/TestExpects/LauncherConfiguration.yml]: >0 [](start = 24, length = 1)

0  is the default value ? default 0 will cause none candidate node selected.

File: frameworklauncher/src/test/resources/TestExpects/PlatformSpecificParametersDescriptor.yml [hunk 2]
```diff
@@ -24,6 +24,7 @@ generateContainerIpList: false
 killAllOnAnyCompleted: false
 killAllOnAnyServiceCompleted: false
 queue: testString
+skipLocalTriedResource: false
```
Comment (qinchen123) [inline frameworklauncher/src/test/resources/TestExpects/PlatformSpecificParametersDescriptor.yml]: >kipLocalTriedResource: fals [](start = 1, length = 27)

the default value need to be true.

File: frameworklauncher/src/test/resources/TestExpects/LauncherConfiguration.yml (hunk 1 above)
Comment (yqwang-ms) [reply frameworklauncher/src/test/resources/TestExpects/LauncherConfiguration.yml]: It is just UT yaml

---
In reply to: [193963658](https://github.com/Microsoft/pai/pull/685#discussion_r193963658) [](ancestors = 193963658)

File: frameworklauncher/src/test/resources/TestExpects/PlatformSpecificParametersDescriptor.yml (hunk 2 above)
Comment (yqwang-ms) [reply frameworklauncher/src/test/resources/TestExpects/PlatformSpecificParametersDescriptor.yml]: It is just UT yaml

---
In reply to: [193963799](https://github.com/Microsoft/pai/pull/685#discussion_r193963799) [](ancestors = 193963799)

**Pull request merged**